### DISCUSSION TITLE
Reference supported Dynatrace version in release

### DISCUSSION THREAD

**Discussion by kristofre** [Dynatrace/dynatrace-configuration-as-code#234] [Ideas]
Since monaco uses different semantic versioning than Dynatrace, it would be good if we could have a reference in the release notes to which Dynatrace version this was tested on to avoid confusion. When fixes are made in monaco to target a particular API behaviour change on Dynatrace side, this should also be noted in the notes.

**Comment by tatomir146** [comment]
@CruzanCaramele I think we could add this to automated release notes generation you are working on?

**Reply by CruzanCaramele** [reply]
yes i think it makes sense to have this included in Release notes

**Reply by tatomir146** [reply]
@CruzanCaramele could we also add this mapping to github pages?
